github3/repos/repo.py

```diff
@@ -1264,6 +1265,61 @@ def ignore(self):
         json = self._json(self._put(url, data=dumps({'ignored': True})), 200)
         return self._instance_or_null(Subscription, json)
 
+    @requires_auth
+    def imported_issue(self, imported_issue_id):
+        """Retrieve imported issue specified by imported issue id.
+
+        :param int imported_issue_id: (required) id of imported issue
+        :returns: :class:`Imported Issue <github3.repos.
+            issue_import.ImportedIssue>`
+        """
+        url = self._build_url('import/issues', imported_issue_id,
+                              base_url=self._api)
+        data = self._get(url, headers=ImportedIssue.IMPORT_CUSTOM_HEADERS)
+        json = self._json(data, 200)
+        return self._instance_or_null(ImportedIssue, json)
+
+    @requires_auth
+    def import_issue(self, title, body, created_at, assignee=None,
+                     milestone=None, closed=None, labels=None, comments=None):
+        """Import issue into this repository.
```

Comment:
> One last nit on this PR: Can we change this to be "Import an issue into the repository."
> 
> Also can we add a link to the documentation gist here and describe how you might use this?
> 
> Cheers!

@sigmavirus24 Will update docstring and provide an example - thanks for the nit :+1: